Pipfile.lock

```diff
@@ -147,12 +155,6 @@
             ],
             "version": "==3.0.4"
         },
-        "ciso8601": {
-            "hashes": [
-                "sha256:bdbb5b366058b1c87735603b23060962c439ac9be66f1ae91e8c7dbd7d59e262"
-            ],
-            "version": "==2.1.3"
-        },
```

Comment:
On my computer we need it!
I think it would be useful to add "ciso8601" in Pipfile.

Reply:
as discussed, you will create a PR for this.